glow.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: glow
+  version: 1.5.1
+  epoch: 0
+  description: Render markdown on the CLI, with pizzazz!
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/charmbracelet/glow
+      tag: v${{package.version}}
+      expected-commit: ad21129703d0bd48a9b45227f56d2737623db57b
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.17.0 golang.org/x/crypto@v0.17.0
+      replaces: github.com/dgrijalva/jwt-go=github.com/golang-jwt/jwt/v5@v5.2.0 github.com/golang-jwt/jwt=github.com/golang-jwt/jwt/v5@v5.2.0
```

Comment:
I was not able to fix it up even after the replace block. 
```bash
[sdk] ❯ wolfictl scan ./packages/x86_64/glow-1.5.1-r0.apk 
🔎 Scanning "./packages/x86_64/glow-1.5.1-r0.apk"
└── 📄 /usr/bin/glow
        📦 github.com/dgrijalva/jwt-go v3.2.0+incompatible (go-module)
            High CVE-2020-26160 GHSA-w73w-5m7g-f7qc
```

Reply:
fixed via https://github.com/kranurag7/os/commit/02fab311454c53d3ea7207b42973c867210cd5e7